rskj-core/src/main/java/co/rsk/net/StatusResolver.java

```diff
@@ -40,7 +40,7 @@ public StatusResolver(BlockStore blockStore, Genesis genesis) {
      */
     public Status currentStatus() {
         Status status;
-        if (blockStore.getMinNumber() != 0) {
+        if (blockStore.getMinNumber() < 0) { // TODO(iago) undo, just for testing
```

Comment:
Just a trick to allow nodes full sync with nodes that have no genesis, to be removed.